packages/jamf_protect/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.1.1"
```

```suggestion
- version: "0.2.0"
```
We increment the minor for enhancement. Same change in manifest.yml